src/utils.test.ts

```diff
@@ -153,4 +136,31 @@ describe('utils', () => {
             expect(fs.writeFile).toBeCalledWith(file, content);
         });
     });
+
+    describe.concurrent('getBufferHash', () => {
+        const testData: [string, string][] = [
+            // [string, sha256 of string]
+            ['test data 1', '05e8fdb3598f91bcc3ce41a196e587b4592c8cdfc371c217274bfda2d24b1b4e'],
+            ['test data 2', '26637da1bd793f9011a3d304372a9ec44e36cc677d2bbfba32a2f31f912358fe'],
+            ['test data 3', 'b2ce6625a947373fe8d578dca152cf152a5bd8aeca805b2d3b1fb4a340e1a123'],
+            ['test data 4', '1e2b98ff6439d48d42ae71c0ea44f3c1e03665a34d1c368ac590aec5dadc48eb'],
+            ['test data 5', '225b2e6c5664bb388cc40c9abeb289f9569ebc683ed4fdd76fef8421c32369b5'],
+        ];
+
+        it.each(testData)('should generate a correct hash for "%s"', (data, hash) => {
+            const calculatedHash = utils.getBufferHash(Buffer.from(data));
+            expect(calculatedHash).toEqual(hash);
+        });
+    });
+
+    describe.concurrent('base64ToArrayBuffer', () => {
+        const strings = ['vite-svg-2-webfont', 'test-string-1', 'test-string-2'];
+
+        it.each(strings)('should match "%s"', string => {
+            const stringAsBase64 = btoa(string);
```

Comment:
Can we use `Buffer.from(string).toString('base64')`?
```suggestion
            const stringAsBase64 = Buffer.from(string).toString('base64');
```